ghost/core/core/server/services/members/members-api/repositories/MemberRepository.js

```diff
@@ -366,7 +369,8 @@ module.exports = class MemberRepository {
                             email: newMember.get('email'),
                             name: newMember.get('name'),
                             source,
-                            timestamp
+                            timestamp,
+                            memberStatus: 'free'
```

Comment:
```suggestion
                            status: 'free'
```

Thinking we should keep this as just `status`, dropping the `member` prefix, since we don't include it for anything else (except the Id...)